docs/docs/learn/4_dev_env_setup/1_intro.md

```diff
@@ -30,7 +33,94 @@ The Aztec Sandbox is particularly important because it gives you a safe, local e
 
 Before we begin, make sure you have:
 
-- **Node.js** (version 22 or later)
-- **Docker** (for running the Sandbox)
-- A code editor (we recommend VS Code for the best Noir support)
-- Basic familiarity with the command line
+- [**Node.js**](https://nodejs.org/en) (version 22 or later)
+- [**Docker**](https://www.docker.com/) (for running the Sandbox)
+- A code editor (we recommend [VS Code](https://code.visualstudio.com/) for the best Noir support)
+- Basic familiarity with the CLI (command line interface)
+
+## What is the sandbox?
+
+The Sandbox is an local development Aztec network running fully on your machine, and interacting with a development Ethereum node. You can develop and deploy on it just like on a testnet or mainnet (when the time comes). The sandbox makes it faster and easier to develop and test your Aztec applications.
+
+What's included in the sandbox:
+
+- Local Ethereum network (Anvil)
+- Deployed Aztec protocol contracts (for L1 and L2)
+- A set of test accounts with some test tokens to pay fees
+- Development tools to compile contracts and interact with the network (`aztec-nargo` and `aztec-wallet`)
+
+All of this comes packages in a Docker container to make it easy to install and run.
+
+## Getting started with the sandbox
```

Comment:
This section is redundant with what is on the next page. We could remove the installation section from this page, or remove it from included code on the next page.